spec/api/api_put_spec.rb

```diff
@@ -0,0 +1,167 @@
+require_relative '../spec_helper'
+require_relative 'api_client.rb'
+require 'securerandom'
+
+RSpec.describe 'PUT method checking to the any data in the body' do
+	api_cl = ApiClient.new
+	body = api_cl.generate_random_body
+	
+	before(:all) do
+		api_cl.create_user(body)
+	end
+	
+	# Positive Functional Testing
+	
+	it 'verifies that username can be edited' do
+		body[:username] = 'KingOfTheNorth'
+		response = api_cl.update_user(body[:username], body)
+		expect(response.status).to eq(200)
```

Comment:
is this check of code enough to say user got edited? maybe better approach is to 'get' this user and check if info got changed? applies to all checks